src/spinnaker-dependencies.yml

```diff
@@ -1,7 +1,7 @@
 versions:
 # spinnaker libs
   fiat: "0.49.8"
-  kork: "3.2.3"
```

Comment:
Do we need to bump the `aws` version here as well?

Reply:
'here' as in this yaml file.

Reply:
No, I only updated it in the template file because it had been bumped in the `yaml` already, and were out of sync

Reply:
OK good catch.